### PR TITLE
feat: login logic added, not found page added

### PR DESCRIPTION
**Pull request by JowelTisso**
* This PR is for the login screen logic
* login and logout logic has been added
* Not found page has also been added

### CODE REVIEW

**Review by Mandhularajitha — APPROVED** [review]
I liked the way you made your code modular,Code looks clean

**Review by ragini07 — APPROVED** [review]
Rest looks code
Clean code,easy to understand naming convention
Colour used in UI looks good

File: src/App.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
   --secondary-color: #fff;
   --txt-color: #424b54;
   --border-color: rgba(148, 161, 179, 0.5);
```
Comment (ragini07) [inline src/App.css:7]: Try using either color code or RGB for consistency.

File: src/App.css [hunk 2]
```diff
@@ -5,6 +5,7 @@
...
+  --danger-color: #f14b4b;
```
Comment (ragini07) [inline src/App.css:8]: Try using either color code or RGB for consistency.

**Review by ragini07 — COMMENTED** [review]
Modular code

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Classnames are self explanatory

File: src/App.css (hunk 2 above)
Comment (JowelTisso) [reply src/App.css:8]: ok

**Pull request merged**